easybuild/easyconfigs/d/deepTools/deepTools-3.1.2-foss-2018a-Python-2.7.14.eb

```diff
@@ -0,0 +1,61 @@
+
+easyblock = "PythonPackage"
+
+name = 'deepTools'
+version = '3.1.2'
+
+homepage = 'https://pypi.python.org/pypi/deepTools/'
+description = """Tools to process and analyze deep sequencing data."""
+
+toolchain = {'name': 'foss', 'version': '2018a'}
+
+source_urls = [PYPI_SOURCE]
+sources = [SOURCE_TAR_GZ]
+checksums = ['fc8b17f0a791b5ba6a3d6bcb32ec0731614da2397d38218217ec3d3cd6ee6129']
+
+python = 'Python'
+pythonversion = '2.7.14'
+pyshortver = ".".join(pythonversion.split(".")[:-1])
+
+versionsuffix = "-%s-%s" % (python, pythonversion)
+
+dependencies = [
+    (python, pythonversion),
+    ('matplotlib', '2.1.2', versionsuffix),
+    ('Pysam', '0.14.1', versionsuffix),
+    ('cURL', '7.58.0'),
+]
+
+exts_defaultclass = 'PythonPackage'
+exts_list = [
+    ('plotly', '2.0.0', {
+        'source_tmpl': '%(name)s-%(version)s.tar.gz',
+        'source_urls': ['https://files.pythonhosted.org/packages/f9/e3/cbd3af94e9567095a9db8c45ae34c7d12599bdeaee521c3f0cb06e6eefb4/'],
+        'checksums': ['eede6581abe00f319909fa84e99f6033b9e735427fb64b570dfbc312fc0dd771'],
+    }),
+    ('py2bit', '0.3.0', {
+        'source_tmpl': '%(name)s-%(version)s.tar.gz',
+        'source_urls': ['https://files.pythonhosted.org/packages/53/bb/547a927bed736ead3dc909e1e552d57c9034bb9493eff80544c0cf6e4828/'],
```

Comment:
@ebirn Use this instead: `https://pypi.python.org/packages/source/p/py2bit/` (+ equivalent for the others)?

The EasyBuild framework has logic to translate this to the URL that includes the hash since EasyBuild v2.8.0, see https://github.com/easybuilders/easybuild-framework/pull/1749 .

The long URLs are annoying & ugly, and have to be changed every time the version changes...